src/main/groovy/com/ofg/twitter/place/extractor/PlacesExtractor.groovy

```diff
@@ -47,6 +54,22 @@ class PlacesExtractor {
         }
         return Optional.empty()
     }
+}
 
-
+//TODO: Remove when integrated with micro-infra-spring (and released)
+@CompileStatic
+@PackageScope
+class CorrelationIdUpdater2PoC {
+    static <T> Closure<T> closureWithId(Closure<T> closure) {
```

Comment:
Maybe. It is important to distinguish between [`withId`](https://github.com/4finance/micro-infra-spring/blob/master/micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/correlationid/CorrelationIdUpdater.groovy#L38)